src/components/maps/MapCords.tsx

```diff
@@ -97,9 +96,8 @@ export default function App({
         ) : null}
 
         <ImageMapper
-          //this is needed to make sure it rerenders when the office selected in the floor changes
           key={marked_position_x + " " + marked_position_y}
-          src={showGroundFloorImg ? GROUND_FLOOR_IMG_URL : imageURL}
+          src={showGroundFloorImg ? GROUND_FLOOR_IMG_URL : imageURL||""}
```

Comment:
its safer to `imageURL ?? ""`. If `imageURL` returns null for some reason, the above will accept `null`. Funny js moment where if it was `"" || null`, `""` would be accepted
![image](https://github.com/open-source-uom/myuom/assets/50584606/ab5ddf2d-b714-49cd-82f9-87ac6367d795)